ui/src/app/components/help-sidebar-icons.tsx

```diff
@@ -94,7 +94,8 @@ export type SidebarIconId =
   | 'notebooksHelp'
   | 'dataDictionary'
   | 'annotations'
-  | 'runtime'
+  | 'runtimeConfig'
```

Comment:
runtimeConfig is a better name for the old version.  It will eventually bee replaced by apps.